packages/openapi-generator/src/parser/schema.spec.ts

```diff
@@ -177,6 +225,23 @@ describe('parseSchema', () => {
     ).toEqual('Property Description');
   });
 
+  it('parses schema object properties', () => {
+    const schemaProperties: OpenAPIV3.SchemaObject = {
+      deprecated: true,
+      example: 100,
+      minimum: 10,
+      maximum: 1000,
+      default: 10
+    };
+    expect(parseSchemaProperties(schemaProperties)).toEqual({
+      deprecated: true,
+      example: 100,
+      minimum: 10,
+      maximum: 1000,
+      default: 10
+    });
+  });
```

Comment:
This test only covers the happy path, but I think it would be good to see that unknown properties are not added for example.